lib/emailnotifications_queue_handler.ts

```diff
@@ -182,12 +184,12 @@ async function handleNotification(
     from: "no-reply@italia.it",
     headers: {
       "X-Italia-Messages-MessageId": messageId,
-      "X-Italia-Messages-NotificationId": notification.id
+      "X-Italia-Messages-NotificationId": notificationId
     },
-    html: messageContent.bodyShort,
+    html: bodyHtml,
     messageId,
     subject: "Un nuovo avviso per te.",
-    text: messageContent.bodyShort,
+    // text: messageContent.bodyMarkdown,
```

Comment:
how to handle this one ?

Reply:
I was considering two options:

* processing the generated HTML with [this](https://www.npmjs.com/package/html-to-text)
* strip the Markdown tokens with something like [this](https://www.npmjs.com/package/remove-markdown)

I'm currently inclined to implement the first option as it will do nice things like re-flowing the text to 80 columns and formatting tables.